package.json

```diff
@@ -218,6 +218,7 @@
     "eslint-stats": "^1.0.0",
     "fast-levenshtein": "^2.0.6",
     "file-loader": "^1.1.11",
+    "local-storage-fallback": "^4.1.1",
     "modernizr": "^3.6.0",
```

Comment:
This module inspects the browser and exports a data structure that always follows the localStorage API but may use a different storage method internally (such as cookies, or in-memory) depending on what features are available in the browser.